test_sims.yml

```diff
@@ -0,0 +1,235 @@
+
+
```

Comment:
All of the test sims, both from the test/ directory and the trick_sims/ directory, are in the yml file at the root of the project. Do yall like that? I could also possibly have 2 separate yml files that live in the test/ and trick_sims/ directories with all their respective sims

Reply:
I like the sims all defined in one spot, although separating them might make sense if there's a difference in how you want to test them in CI.  Y'all know more about the Trick-specific workflow and what makes sense so my opinion isn't super relevant 😝